proto/vtctldata.proto

```diff
@@ -1984,6 +1984,7 @@ message WorkflowSwitchTrafficRequest {
   bool dry_run = 9;
   bool initialize_target_sequences = 10;
   repeated string shards = 11;
+  bool force = 12;
```

Comment:
This request keeps getting bigger. I wish we had implemented something similar to workflow options.

Reply:
WorkflowOptions are persisted in the workflow record as they affect the workflow generally. This flag only applies to a single switch traffic call and shouldn’t be persisted.

Reply:
I’m also no sure what’s bad about adding command flags and the corresponding proto fields for it?

Reply:
If this related to the actual wire protocol message size, then AFAIK embedding doesn't significantly impact that one way or the other. If that it is the concern though, we could move all fields to optional so that they are nil pointers instead of zero value structs. I recently added one new optional field and moved some existing ones to optional here: https://github.com/vitessio/vitess/pull/16734

I can see e.g. that all of the (non-repeated) fields seem to be optional now in k8s: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/core/v1/generated.proto